README.adoc

```diff
@@ -42,37 +45,49 @@ and configure a _Run/Debug configuration_ like this:
 * Working directory: `$MODULE_DIR$` (must resolve to optaplanner-examples directory)
 * Use classpath of module: `optaplanner-examples`
 
-=== Starter issues
+== Contributing to OptaPlanner
+
+This is an open source project, and you are more than welcome to contribute :heart:!
+
+=== Getting into
 
 If you're just starting out with OptaPlanner and want to contribute,
 take a look at our https://issues.redhat.com/issues/?jql=project%20%3D%20PLANNER%20AND%20status%20in%20(Open%2C%20Reopened)%20AND%20labels%20%3D%20starter%20ORDER%20BY%20priority%20DESC[starter issues].
 They're specifically chosen to be easier for first time contributors.
 
-== Developing Drools, OptaPlanner and jBPM
+If you want to contribute or start an opinionated discussion, join our https://groups.google.com/g/optaplanner-dev[discussion] or send an e-mail directly to optaplanner-dev@googlegroups.com.
+
+If you want to submit an issue, please check out the https://issues.redhat.com/projects/PLANNER/issues[OptaPlanner Jira project].
 
-*If you want to build or contribute to a kiegroup project, https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/main/README.md[read this document].*
+=== Time to make a change?
 
-*It will save you and us a lot of time by setting up your development environment correctly.*
-It solves all known pitfalls that can disrupt your development.
-It also describes all guidelines, tips and tricks.
-If you want your pull requests (or patches) to be merged into main, please respect those guidelines.
+Every change must go through *Pull Request*. Please follow the <<Contributing>> section for more details. OptaPlanner has a CI in place that runs checks against your branch after you submit the PR. Don't worry if something gets wrong or breaks other parts of functionality; our friendly PR maintainers will help you with that.
 
-=== Code style
+=== Contributing
 
-OptaPlanner has adopted the https://github.com/quarkusio/quarkus[Quarkus] code style, enforces it, and automatically formats code during the build.
-To setup your IDE, please see the
-<<build/optaplanner-ide-config/ide-configuration.adoc#, IDE Setup Instructions>>.
+1. Fork it (https://github.com/kiegroup/optaplanner)
+2. Create your feature branch (`git checkout -b feature/fooFeature`)
+3. Commit your changes (`git commit -am 'Add some fooFeature'`)
+4. Push to the branch (`git push origin feature/fooFeature`)
+5. Create a new Pull Request
```

Comment:
```suggestion
. Create a new pull request.
```

Reply:
@dupliaka +1 to this.

* No need to capitalize pull request. It's part of the GitHub jargon (branch, fork, workflow, ...) and not a proper name (like GitHub, OptaPlanner, Maven, etc.).
* This list items are full sentences so they should end with a period.

Reply:
Fixed